package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "types": "index.d.ts",
   "dependencies": {
-    "underscore": "~1.7.0",
+    "underscore": "^1.12.1",
```

Comment:
Because you're installing `"underscore": "^1.12.1",` and there isn't a `package-lock.json` or `yarn.lock` you'll actually be installing the latest minor release. 

Check out your `npm` packages locally and you should see `"version": "1.13.1",` located at, `node_modules/underscore/package.json`

Reply:
Yeah, you are right! We would be using 1.13.1 in this case. Thanks for the clarification!